service-python/pdfgenerator/src/lib/templates/hypertension-v2.html

```diff
@@ -306,7 +306,23 @@ <h2 class="m-0 mb-13"><b>Schedular evidence for hypertension</b></h2>
             <td {% if medication.organization == "" and medication.dataSource == "MAS" %}class="empty" {% endif %}>{% if not medication.organization and medication.dataSource == "MAS" %}n.a.{% elif not medication.organization and medication.dataSource == "LH" %} VAMC record {% elif medication.organization and medication.dataSource == "LH" %} VAMC record - {{ medication.organization }}{% else %}{{ medication.organization }}{% endif %}</td>
             <td {% if medication.receiptDate == "" %}class="empty" {% endif %}>{% if medication.receiptDate == ""%}n.a.{% else %}{{ medication.receiptDate }}{% endif %}</td>
             <td {% if medication.page == "" %}class="empty" {% endif %}>{% if medication.page == ""%}n.a.{% else %}{{ medication.page }}{% endif %}</td>
-            <td {% if medication.document == "" %}class="empty" {% endif %}>{% if medication.document == ""%}n.a.{% else %}{{ medication.document }}{% endif %}</td>
+            <td {% if medication.document == "" %}class="empty" {% endif %}>
+              {% if medication.dataSource == "MAS" %}
+                {% if medication.organization == "VAMC Other Output Reports" and medication.document == "" %}
+                  Pending upload to VBMS eFolder
+                {% elif medication.document == "" %}
+                  n.a.
+                {% else %}
+                  {{ medication.document }}
+                {% endif %}
+              {% else %}
+                {% if medication.document == ""%}
+                  n.a.
+                {% else %}
+                  {{ medication.document }}
```

Comment:
I believe we need to implement this logic for BP readings as well as medication. I am not sure if it is needed for conditions.